### PR TITLE
Implement airburst support to Bullet.

### PR DESCRIPTION
**Pull request by GraionDilach**
Has nothing to do with the regression #12628 intended to fix.

### CODE REVIEW

File: OpenRA.Mods.Common/Projectiles/Bullet.cs [hunk 1]
```diff
@@ -142,6 +145,11 @@ public Bullet(BulletInfo info, ProjectileArgs args)
 				target += WVec.FromPDF(world.SharedRandom, 2) * maxOffset / 1024;
 			}
 
+			if (info.AirburstAltitude > WDist.Zero)
+			{
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Projectiles/Bullet.cs]: You should drop the braces.

**Review by reaperrr — APPROVED** [review]
👍 after @abcdefg30's style nit is fixed.

**Comment by GraionDilach** [comment]
Ah, I totally forgot this one. Fixed now.

**Comment by reaperrr** [comment]
@abcdefg30 (or anyone else who reads this): The style nit was adressed and this is quite easy to review, so it would be nice to get it merged soon. Just add `AirburstAltitude: 1c0` to any artillery in the mods and watch (I recommend D2k Siege Tank, because the larger projectile shadow and brighter ground make it easier to see).

**Comment by ltem** [comment]
:+1: it works 

image
image

**Comment by atlimit8** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/709c39aec00936ed8f81207249c6a35a75100407...577031c35e9aeb9390775998146228be898533d7)

**Pull request merged**